src/styles/createTheme.tsx

```diff
@@ -8,6 +8,15 @@ const green = '#1D878C'
 
 const theme: Record<string, any> = createTheme({
   spacing: 5,
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 900,
+      lg: 1024,
+      xl: 1536,
```

Comment:
xl - is also default value

Reply:
Typescript provides an error if I use remove it there.
<img width="574" alt="image" src="https://github.com/WeFriiends/WeFriiendsWeb/assets/324031/a1f57679-a851-4307-8ae8-33fa24d5275a">

Reply:
Ok, understood